core/translations/signatures.json

```diff
@@ -1,8 +1,8 @@
 {
   "current": {
-    "merkle_root": "70b288d1c7cc176d116e3fb4f0737daba7c465b6ee616278396843933d69afd3",
-    "datetime": "2024-11-04T07:30:55.564229",
-    "commit": "dfdd44becba40dbdff15e8aa88edbf6cedf063ea"
+    "merkle_root": "3af4b3858cde58d36f884fb0ea592d488c8380baaed2241941bcdb38276d8d25",
+    "datetime": "2024-11-06T13:24:51.443025",
+    "commit": "9c32e76315450f636bde59a25da2b2ff11699682"
   },
   "history": [
```

Comment:
while you're at this, please add the signatures that we generated recently

Reply:
sure - what about the duplicate versions, append or overwrite?

Reply:
i think they're appended, the signatures are for a different commit.
the `translations/cli.py sign` command should do the right thing, i believe

Reply:
pushed ee3589b777498448037d476fa0247770be17fa41, please check